<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BMSClass</class>
 <widget class="QMainWindow" name="BMSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>BMS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/BMS图标.ico</normaloff>../Resources/BMS图标.ico</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{background-image: url(:/登录界面的固定图片/Resources/固定/背景1.jpg);}

#remberPW,#lgFindPW,#lgRegister
{
font: 12pt &quot;Adobe 黑体 Std R&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">image: url(:/登录界面的固定图片/Resources/登录界面背景.jpg);</string>
   </property>
   <widget class="HeadPor" name="lgFramePNG">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="lgID">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>261</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>261</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lgLableID">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../BMS.qrc">:/登录界面的固定图片/Resources/小按钮/用户名.ico</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lgPW">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="lgLablePW">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../BMS.qrc">:/登录界面的固定图片/Resources/小按钮/密码.ico</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="remberPW">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QToolButton" name="lgFindPW">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>找回密码</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lgOK">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 23pt &quot;Adobe 黑体 Std R&quot;;
background-image: url(:/登录界面的固定图片/Resources/固定/登录按钮.jpg)</string>
    </property>
    <property name="text">
     <string>登         录</string>
    </property>
   </widget>
   <widget class="QToolButton" name="lgRegister">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>469</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="lgSet">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset resource="../BMS.qrc">
      <normaloff>:/登录界面的固定图片/Resources/小按钮/设置.ico</normaloff>:/登录界面的固定图片/Resources/小按钮/设置.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="lgMininum">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>最小化</string>
    </property>
    <property name="icon">
     <iconset resource="../BMS.qrc">
      <normaloff>:/登录界面的固定图片/Resources/小按钮/最小化.ico</normaloff>:/登录界面的固定图片/Resources/小按钮/最小化.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="lgClose">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="icon">
     <iconset resource="../BMS.qrc">
      <normaloff>:/登录界面的固定图片/Resources/小按钮/退出.ico</normaloff>:/登录界面的固定图片/Resources/小按钮/退出.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="lgShowPW">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeadPor</class>
   <extends>QFrame</extends>
   <header location="global">headpor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../BMS.qrc"/>
 </resources>
 <connections/>
</ui>
